VisualizationBase/src/Scene.cpp

```diff
@@ -606,6 +606,7 @@ void Scene::removeAllOnZoomHandlers()
 int Scene::addOnZoomHandler(OnZoomHandler onZoomHandler, OnZoomHandlerRemove onZoomHandlerRemove)
 {
 	onZoomHandlers_.insert(addOnZoomHandlerId_, {onZoomHandler, onZoomHandlerRemove});
+	onZoomHandler(mainViewScalingFactor_);
```

Comment:
I added this to have the right scaling of components immediately after adding their OnZoomHandler. Without this change it would only take effect after the next zooming.